src/Bootstrap/src/AutoConfiguration/BootstrapScanner.cs

```diff
@@ -256,12 +256,12 @@ private void WireDistributedTracing()
         _logger.LogInformation("Configured distributed tracing");
     }
 
-    private void WireCloudFoundryContainerIdentity()
+    private void WireAppInstanceIdentity()
     {
-        _wrapper.ConfigureAppConfiguration(configurationBuilder => configurationBuilder.AddCloudFoundryContainerIdentity());
-        _wrapper.ConfigureServices(services => services.AddCloudFoundryCertificateAuth());
+        _wrapper.ConfigureAppConfiguration(configurationBuilder => configurationBuilder.AddAppInstanceIdentityCertificate());
+        _wrapper.ConfigureServices(services => services.ConfigureCertificateOptions("AppInstanceIdentity", null));
```

Comment:
I wonder if this should become:
```c#
        _wrapper.ConfigureServices(services =>
        {
#if !NET6_0
            services.AddAuthorizationBuilder().AddAppInstanceIdentityCertificate();
#endif
        });
```

If yes, then we need a conditional dependency in the project file:
```xml
<ProjectReference Condition="'$(TargetFramework)' != 'net6.0'" Include="..\..\..\Security\src\Authorization.Certificate\Steeltoe.Security.Authorization.Certificate.csproj" PrivateAssets="All" />
```

Reply:
I didn't want to add the auth pieces since we won't be able to control the way middlewares are ordered... I think the better options is to take everything certificate-related out of the bootstrap project

Reply:
I agree, thanks.